contracts/middle-layer/resource-mirror/MultiMessage.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "../../interface/ICrossChain.sol";
+import "../../interface/IMultiMessage.sol";
+import "./CmnHub.sol";
+import "./storage/MultiStorage.sol";
+
+contract MultiMessage is MultiStorage, CmnHub, IMultiMessage {
+    uint8 public constant ACK_PACKAGE = 0x01;
+    uint8 public constant FAIL_ACK_PACKAGE = 0x02;
+
+    mapping(address => bool) public whitelistTargets;
+
+    constructor() {
+        _disableInitializers();
+    }
+
+    /*----------------- initializer -----------------*/
+    function initialize(address _ERC721_token, address _additional) public initializer {
```

Comment:
Multi message dont have `_ERC721_token` and `_additional`

Reply:
fixed